app/schemas/pagination.js

```diff
@@ -0,0 +1,28 @@
+const errors = require('../constants/errors');
+
+exports.paginationSchema = {
+  limit: {
```

Comment:
Limit and page are optional, not required. If both exist in request, you must check that both are integers